src/vma/proto/dst_entry.h

```diff
@@ -98,6 +98,7 @@ class dst_entry : public cache_observer, public tostr, public neigh_observer
 	void		return_buffers_pool();
 	int		get_route_mtu();
 	inline header*	get_network_header() { return &m_header;}
+	inline ring*	get_ring() { return m_p_ring;}
```

Comment:
Why to expose the ring if we only want to the tx channel fd ?

Reply:
more generic for future needs.
ring can be null